### PR TITLE
Binomial coefficient in python 3

### PR DESCRIPTION
**Pull request by Thuva4**

### CODE REVIEW

File: code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//binomial coefficient in python 3
```
Comment (yatharthshahjpr) [inline code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py]: Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

File: code/mathematical-algorithms/factorial/factorial.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//factorial without using recursion
```
Comment (yatharthshahjpr) [inline code/mathematical-algorithms/factorial/factorial.py]: Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

**Comment by AdiChat** [comment]
Kindly remove `factorial.py` as it already exists :+1:

**Comment by Thuva4** [comment]
@AdiChat  I did not see a factorial function.py without recursion in the folder. Sorry, If I am wrong.

File: code/mathematical-algorithms/factorial/factorial.py (hunk 2 above)
Comment (arnavb) [reply code/mathematical-algorithms/factorial/factorial.py]: @yatharthshahjpr The comment needs to start with a `#`

File: code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py (hunk 1 above)
Comment (arnavb) [reply code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py]: @yatharthshahjpr The comment needs to start with a #

**Comment by AdiChat** [comment]
Yes, computing factorial by iteration exists ([see](https://github.com/OpenGenus/cosmos/blob/master/code/mathematical-algorithms/factorial/factorial_iteration.py)) 👍 

Your code `BinomialCoefficient.py` has been merged in your another pull request 👍

**Pull request closed without merging**